app/models/endpoint.rb

```diff
@@ -70,11 +70,6 @@ def self.seed_archive_endpoints_from_config(preservation_policies)
     end
   end
 
-  # TODO: move to EndpointType class?  e.g. .default_for_storage_root
-  def self.default_storage_root_endpoint_type
-    EndpointType.find_by!(type_name: Settings.endpoints.storage_root_defaults.endpoint_type_name)
-  end
-
```

Comment:
See ticket #215 -- I raised this question a while back.

Reply:
sorry it took me so long to see the error of my ways and circle back 🙂